tests/integration/test_analysisd/test_rootcheck/test_configuration/test_rootcheck_delete.py

```diff
@@ -0,0 +1,145 @@
+'''
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The 'rootcheck' tool allows to define policies in order to check if the agents
+       meet the requirement specified. The rootcheck engine can check if a process is running, if a file is
+       present and if the content of a file contains a pattern,
+       or if a Windows registry key contains a string or is simply present.
+
+components:
+    - rootcheck
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-analysisd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/policy-monitoring/rootcheck
+    - https://documentation.wazuh.com/current/user-manual/reference/daemons/wazuh-analysisd.html
+
+tags:
+    - rootcheck
+'''
+
+import pytest
+import time
+from pathlib import Path
+
+from wazuh_testing.constants.paths.sockets import WAZUH_DB_SOCKET_PATH
+from wazuh_testing.tools.socket_controller import SocketController
+from wazuh_testing.utils import configuration
+from wazuh_testing.utils.services import control_service
+from conftest import retrieve_rootcheck_rows
+
+from . import CONFIGS_PATH, TEST_CASES_PATH
+
+# Marks
+pytestmark = [pytest.mark.linux, pytest.mark.tier(level=0), pytest.mark.server]
+
+def send_delete_table_request(agent_id):
```

Comment:
Create a generic function in the framework and use it instead of this. I think this function should be in https://github.com/wazuh/qa-integration-framework/blob/main/src/wazuh_testing/utils/sockets.py.